setup.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name='OpenFisca-Web-API-Rewrite',
+    version='0.1.0',
+    author='OpenFisca Team',
+    author_email='contact@openfisca.fr',
+    description=u'POC of a new API for OpenFisca',
```

Comment:
That kind of stuff stays forever. Don't say it's a PoC: this is a transient property that the version number should carry. State the goal: this is the OpenFisca API.